distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc

```diff
@@ -271,21 +271,33 @@ image::search-results-options.png[search results options]
 * *Expand Metacard View*: Navigates to a view that only focuses on this particular result.
 * *Create Search from Location*: Searches for all records that intersect the current result's location geometry.
 
-==== Bookmarks Tab
+==== Lists Tab
 
-Bookmarked results are denoted with a yellow *star* (image:bookmarked-icon.png[]) icon.
+Results can be added to lists to organize and perform actions on a set of results.
```

Comment:
Maybe this could be clearer as something like "Lists organize results and enable performing actions on those sets of results." ❓

Reply:
Sure I'll change it